include/Ecal/EcalVetoProcessor.h

```diff
@@ -119,6 +134,12 @@ class EcalVetoProcessor : public framework::Producer {
   double avgLayerHit_{0};
   double stdLayerHit_{0};
   double ecalBackEnergy_{0};
+  // MIP tracking
```

Comment:
Add a `///` style comment for each variable explaining what each variable is for.

Reply:
Quick question:  Would the comments be on a separate line, or would they be on the same line as the variables they're referencing?

Reply:
Above the variable they are referencing.